securedrop/create-demo-user.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from sqlalchemy.exc import IntegrityError
+
+os.environ["SECUREDROP_ENV"] = "dev"  # noqa
+import journalist_app
+import config
+from db import db
+from models import Journalist
+
+
+def add_test_user(username, password, otp_secret, is_admin=False):
+    context = journalist_app.create_app(config).app_context()
+    context.push()
+    valid_password = "correct horse battery staple profanity oil chewy"
+
+    try:
+        user = Journalist(username=username,
+                          password=valid_password,
+                          is_admin=is_admin)
+        user.otp_secret = otp_secret
+        user.pw_salt = user._gen_salt()
+        user.pw_hash = user._scrypt_hash(password, user.pw_salt)
```

Comment:
Why are you hashing `password` but not `valid_password`?

Reply:
The Journalist.set_password function is not used because it enforces
seven words diceware passwords and the demo password does not comply.
Although it would make for a more educative example to use a seven
words password, it is more difficult to copy/paste, all pages
displaying the existing password would have to be modified and the
users would need to update.